src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -914,6 +922,68 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // not working properly.
         manager.register_pass<ov::pass::Validate>();
 
+        manager.register_pass<ov::pass::RoPEFusion>(true);
+        pass_config->disable<ov::pass::RoPEFusionGPTJ>();
+        pass_config->disable<ov::pass::RoPEFusionIOSlicing>();
+        pass_config->disable<ov::pass::RoPEShareCosSin>();
+
+        float activations_scale_factor = config.get_property(ov::hint::activations_scale_factor);
+
+        if (activations_scale_factor > 0.f && infer_precision == ov::element::f16 && !enableInt8) {
```

Comment:
Why `!enableInt8` is needed? What if we run a model with hybrid quantization?

Reply:
When `enableInt8` is `true`, activations of Convolution and Matmul are int8. So, I thought that activations scaling cannot be applied in this case. Actually, I met an issue when I tested with a resnet50-int8 model. But, I agree with your comments that we need to support hybrid quantized models. I think we can do it better after `ScaleDownSingleLayer` is replaced with updated LPT passes in the future.

Reply:
As an option, we can move activation scaling pipeline after main LPT and match `ScaleDownSingleLayer` only on nodes which are not in low precision